requirements.txt

```diff
@@ -7,7 +7,7 @@ Django==2.1
 django-cors-headers==2.4.0
 django-cors-middleware==1.3.1
 django-extensions==2.1.4
-django-rest-swagger==2.2.0
+django-rest-swagger==2.1.0
```

Comment:
Your PR description says upgrade django_rest_framework v2.2.0 to v2.2.1. This seems like you downgraded to 2.1.0

Reply:
Updated as required.

Reply:
No it's not. The description says `Downgrade django_rest_framework v2.2.0 to v2.2.1`

Reply:
Check your PR description

Reply:
Sorry about that. I think its fixed now.